modules/integration/src/test/scala/scala/cli/integration/InstallAndUninstallCompletionsTests.scala

```diff
@@ -69,9 +69,7 @@ class InstallAndUninstallCompletionsTests extends ScalaCliSuite {
   lazy val fishRcScript: String = {
     val progName = "scala-cli"
     val script =
-      s"""
-    complete $progName -a '($progName complete fish-v1 (math 1 + (count (__fish_print_cmd_args))) (__fish_print_cmd_args))'
-         |""".stripMargin
+      s"""complete $progName -a '($progName complete fish-v1 (math 1 + (count (__fish_print_cmd_args))) (__fish_print_cmd_args))'"""
```

Comment:
The whitespace is not significant, so this should be fine. I see that there is an extra newline actually generated previously, so I guess this should count as a fix? :sweat_smile:

Reply:
Yeah, I thought the same.